tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -293,7 +294,9 @@ export class GraphExecutor implements FunctionExecutor {
             if (tensor && !tensor.kept && !tensorsToKeep.has(tensor.id)) {
               const count = intermediateTensorConsumerCount[tensor.id];
               if (count === 1) {
-                tensor.dispose();
+                if (!keepTensorForDebug) {
+                  tensor.dispose();
```

Comment:
Do you need to take care of when to dispose() these intermedia tensors?

Reply:
You are right. This is for debugging purpose, so for functionality, I think it is OK.  But If you insist, I can try to add some extra API to do dispose.

Reply:
Add disposeDebugTensors